### PR TITLE
Fixes #1950 - changed style if untriaged issues > 50

### LINKED ISSUE #1950: Triage Dashboard - if untriaged issues > 50, mark the number as red for alert
Easily to know if there is issue flood from sprint?

### PR DESCRIPTION
**Pull request by magsout**
r @karlcow  ?

One question, I added a test for `Needstriage issues` and `Older than 48h`.

Is it needed for  `Older than 48h`?

### CODE REVIEW

File: webcompat/templates/dashboard/triage/stats.html [hunk 1]
```diff
@@ -62,7 +84,7 @@ <h2 class="wc-Stat-title">Needstriage issues</h2>
     </div>
   </article>
   <article class="wc-Stats-article">
-    <div class="wc-Stat">
+    <div class="wc-Stat {% if stats.total > 50 %} wc-Stat--alert {% endif %}">
```
Comment (karlcow) [inline webcompat/templates/dashboard/triage/stats.html]: @magsout Maybe not needed here indeed. This number is always lower or equal to the total. It's a subproduct of the total.

File: webcompat/templates/dashboard/triage/stats.html [hunk 2]
```diff
@@ -33,7 +33,29 @@
...
             justify-content: center;
   }
-  
+
+  .wc-Stat--alert {
+    background-color: #c12600;
+    animation: pulse 5s infinite;
+  }
+
+  @keyframes pulse {
```
Comment (karlcow) [inline webcompat/templates/dashboard/triage/stats.html]: I thought it was cool to have the slight animation too… then… when testing CPU went 30% higher.
@magsout do you experience the same thing?

Comment (magsout) [reply webcompat/templates/dashboard/triage/stats.html]: not 30%, but 15%. although on home of webcompat, only 0.8%, hum weird, it's only a small infinite animation.. So I remove it

**Comment by magsout** [comment]
@karlcow done

**Pull request merged**